Libraries/LibWeb/Painting/Paintable.h

```diff
@@ -175,6 +178,8 @@ class Paintable
     bool m_absolutely_positioned : 1 { false };
     bool m_floating : 1 { false };
     bool m_inline : 1 { false };
+
+    bool m_is_affected_by_non_local_effects { false };
```

Comment:
`is_affected_by_non_local_effects()` isn't actually used anywhere in the change.

Reply:
Good point, that was left over from an earlier iteration, sorry.
I've removed it now, and I'll also try to take some measurements of pages / things that should be improved by this.